packages/contract/src/contract.ts

```diff
@@ -1,8 +1,9 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
 import type { BigNumberish } from '@ethersproject/bignumber';
 import { Logger } from '@ethersproject/logger';
-import type { JsonFragment, FunctionFragment } from '@fuel-ts/abi-coder';
+import type { FunctionFragment } from '@fuel-ts/abi-coder';
```

Comment:
CI is fialing here

Reply:
Whoops. Seems like ESLint autofix on save didn't work.